metadata-transformer/src/main/java/no/unit/nva/metadata/DcTerms.java

```diff
@@ -0,0 +1,85 @@
+package no.unit.nva.metadata;
+
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum DcTerms {
```

Comment:
It was easier to include everything, rather than filter out just what we use right now.